debugger/websockets/packet_queue.c

```diff
@@ -82,7 +81,7 @@ void packet_enqueue (Emulator *emu, void *item, size_t size,
     while (cur->next != NULL) {
       cur = cur->next;
     }
-  
+
     cur->next = tail = (Packet *) calloc(1, sizeof(Packet));
```

Comment:
Missing `free`

Reply:
queue item is freed in packet_dequeue;
queue item message WAS at some point in time freed, but this is commented out (no idea why, but probably didn't work)

We don't use this too much and we don't have tests for it... I prefer to leave it as it